packages/node-opcua-secure-channel/source/security_policy.ts

```diff
@@ -31,11 +31,15 @@ import {
     verifyMessageChunkSignature
 } from "node-opcua-crypto";
 import { EncryptBufferFunc, SignBufferFunc } from "node-opcua-chunkmanager";
+import { make_warningLog } from "node-opcua-debug";
 
 // tslint:disable:no-empty
 function errorLog(...args: any[]) {
     /** */
 }
+const warningLog = make_warningLog(__filename);
+const doTraceChunk = process.env.NODEOPCUADEBUG && process.env.NODEOPCUADEBUG.indexOf("CHUNK") >= 0;
```

Comment:
CHUNK is used for tracing low level packet chunking on the transport layer

Reply:
fixed in commit d413004